### DISCUSSION TITLE
Metadata files applicable to all types of image archives

### DISCUSSION THREAD

**Discussion by RuiDSaaCC** [gotson/komga#1713] [Ideas]
Translate:

This file should be placed in the same directory as the image archive, with the name of the image archive plus the suffix (ABC. pdf>ABC. pdf. ext).
The file is in UTF-8 format and can be directly modified using a text editor.
The file content consists of metadata type+metadata value+metadata multilingual value (optional), for example:

```xml
<Title>name<Title:EN>name<Title:JP>名前<Title:CN>名称</Title>
<Series>Series<Series:EN>Series<Series:JP>シリーズ<Series:CN>系列</Series>
<Number>2.0</Number>
<Writer>ABC<Writer:EN>ABC<Writer:JP>ABC<Writer:CN>ABC</Writer>
<Tags>Monday<Tags:EN>Monday<Tags:JP>月曜日<Tags:CN>星期一</Tags>
<Tags>Tuesday<Tags:EN>Tuesday<Tags:JP>火曜日<Tags:CN>星期二</Tags>
<Tags>Friday<Tags:EN>Friday<Tags:JP>金曜日<Tags:CN>星期五</Tags>
```

The tag function can provide tag group support:

```xml
<TagGroup>Tag Group<TagGroup:EN>Tag Group<TagGroup:JP>ラベルグループ<TagGroup:CN>标签组
  <Tags>Monday<Tags:EN>Monday<Tags:JP>月曜日<Tags:CN>星期一</Tags>
</TagGroup>
```

Provide chapter support:
Locate to a specific page, which is very useful for image archives containing a large number of images

```xml
<Chapter>4835<#>Chapter Title<Chapter:EN>Chapter Title<Chapter:JP>章のタイトル<Chapter:CN>章节标题</Chapter>
```

advantage:
* No longer relying on the metadata options provided by the image archive itself, providing complete metadata options for all types of image archives.
* It will not damage the integrity of the image file itself, which is very helpful for BT or PT.
* Strong scalability, customizable metadata types.
* Support tag multilingual, providing native language translated text for users of different languages (if available)

**Comment by yatounoneko** [comment]
I believe using the existing .opt format would be better. Calibre is already powerful enough to provide detailed editing functions and export metadata files.

Alternatively, by editing through Komga, directly exporting a file named the same "comicsfilename.ComicInfo.xml"(in comics file folder) as the external metadata reference file could be better.
